static/src/js/actions/__tests__/focusedCollection.test.js

```diff
@@ -885,17 +884,18 @@ describe('changeFocusedCollection', () => {
       useEdscStore.setState({
         timeline: {
           getTimeline: getTimelineMock
-        }
-      })
-
-      const store = mockStore({
+        },
         preferences: {
+          ...useEdscStore.getInitialState().preferences,
```

Comment:
I saw this syntax in another file and we should do this instead of the calls to `getInitialState()`

```
    useEdscStore.setState((state) => ({
      ...state,
      preferences: {
        ...state.preferences,
        preferences: {
          ...state.preferences.preferences, 
          granuleSort: 'default'
        }
      }
    }))
```